packages/common/src/ceramic-api.ts

```diff
@@ -95,17 +97,18 @@ export interface AdminApi {
   getIndexedModels(): Promise<Array<StreamID>>
 
   /**
-   * List indexed model streams with fields
+   * List indexed model streams with data
    */
-  getIndexedModelsWithFieldIndices(): Promise<Array<ModelFieldsIndex>>
+  getIndexedModelData(): Promise<Array<ModelData>>
 
   /**
    * Adds model streams to index
    *
-   * @param modelsIDs - array of model stream IDs to add to index
+   * @param modelsIDs - array of model stream IDs to add to index. This parameter is deprecated
+   * and indices should be specified instead
    * @param indices - array of model streams with field indices to index
    */
-  startIndexingModels(modelsIDs: Array<StreamID>, indices?: Array<ModelFieldsIndex>): Promise<void>
+  startIndexingModels(modelsIDs: Array<StreamID>, indices?: Array<ModelData>): Promise<void>
```

Comment:
also, I liked how above we split `getIndexedModels` and `getIndexedModelData` into two functions.  Can we do the same here?  `startIndexingModels` and `startIndexingModelData`?  Then the whole `startIndexingModels` method can just be marked as deprecated, and we don't have to worry about the case where the two args to this function specify a different set of models between them